Install docs spruce-up & VS for Mac and VSCode mentions (#1920)
NuGet/docs.microsoft.com-nuget: docs/install-nuget-client-tools.md

| [Visual Studio for Mac](/visualstudio/mac/nuget-walkthrough) | On Mac, certain NuGet capabilities are built-in directly. Package Manager Console is not presently available. For other capabilities, use the `dotnet.exe` or `nuget.exe` CLI tools.  | [Visual Studio for Mac](https://visualstudio.microsoft.com/vs/mac/) |
| [Visual Studio Code](https://code.visualstudio.com/docs) | On Windows, Mac, or Linux, NuGet capabilities are available through marketplace extensions, or use the `dotnet.exe` or `nuget.exe` CLI tools. | [Visual Studio Code](https://code.visualstudio.com/Download/)|

The [MSBuild CLI](reference/msbuild-targets.md) also provides the ability to restore and create packages, which is primarily useful on build servers. MSBuild is not a general-purpose tool for working with NuGet.

> Package Manager Console commands work only within Visual Studio on Windows and do not work within other PowerShell environments.

Reviewer: Remove `commands` maybe?

```suggestion
Package Manager Console works only within Visual Studio on Windows and does not work within other PowerShell environments.
```

Author: Doesn't make sense. The point is that the commands are different. Can you double-check this?

Reviewer 2: I think the original looks good, tbh

## Visual Studio
### Install on Visual Studio 2017 and newer
Starting in Visual Studio 2017, the installer includes the NuGet Package Manager with any workload that employs .NET. To install separately, or to verify that the Package Manager is installed, run the Visual Studio installer and check the option under **Individual Components > Code tools > NuGet package manager**.

### Install on Visual Studio 2015 and older
NuGet Extensions for Visual Studio 2013 and 2015 can be downloaded from [https://dist.nuget.org/index.html](https://dist.nuget.org/index.html).

For Visual Studio 2010 and earlier, install the "NuGet Package Manager for Visual Studio" extension. Note, if you can't see the extension in the first page of search results, try changing the Sort By dropdown to "Most Downloads", or an alphabetical sort.

> ## CLI tools

Reviewer 3: `### CLI tools`

Reviewer 2: I believe this is actually the correct level for this header.

> A CLI tool is required to support NuGet features in the IDE. You can use either the `dotnet` CLI or the the `nuget.exe` CLI. The `dotnet` CLI is installed with some Visual Studio workloads, such as .NET Core. The `nuget.exe` CLI must be installed separately as described earlier.

Reviewer: > A CLI tool is required to support NuGet features in the IDE.

I don't really follow this...seems counter intuitive.

Author: I didn't write it, I just moved it. Can you suggest a rewrite?

Author: Perhaps:
`To support some NuGet features, a CLI tool is required outside the IDE.`

?

Reviewer 3: may be start the content with - "You can use either the `dotnet` CLI or the the `nuget.exe` CLI to support NuGet features in the IDE."

> A CLI tool is required to support NuGet features in the IDE. You can use either the `dotnet` CLI or the the `nuget.exe` CLI. The `dotnet` CLI is installed with some Visual Studio workloads, such as .NET Core. The `nuget.exe` CLI must be installed separately as described earlier.

Reviewer 4: ```suggestion
You can use either the `dotnet` CLI or the `nuget.exe` CLI to support NuGet features in the IDE. The `dotnet` CLI is installed with some Visual Studio workloads, such as .NET Core. The `nuget.exe` CLI must be installed separately as described earlier.
```
